t/fuzz/fuzz_parseini.c

```diff
@@ -0,0 +1,38 @@
+#include "fio.h"
+
+static int initialized = 0;
+
+int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
+{
+    if (size < 2)
+        return 0;
+
+    if (initialized == 0) {
+        if (fio_init_options()) {
+            printf("Failed fio_init_options\n");
+            return 1;
+        }
+
+        char *fakeargv[] = {(char *) "fuzz",
```

Comment:
Variable declarations go at the top. And this should follow the general coding style, tabs are tabs, pointer '*' goes with the variable name, etc.

Reply:
> Variable declarations go at the top.

Top of the block or the function ?
Changed it to top of the block.

> tabs are tabs, pointer '*' goes with the variable name

This should be good

Reply:
Top of the block is fine.